vaadin-date-picker-flow-parent/vaadin-date-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datepicker/DatePickerIT.java

```diff
@@ -151,181 +123,6 @@ true, executeScript("return arguments[0].max === '1985-01-19'",
-            String inputValue, LocalDate expectedDate) {
-        setInputValueAndAssert(datePicker, inputValue, inputValue,
-                expectedDate);
-    }
-
-    @Test
-    public void selectDatesBeforeYear1000SimulateUserInput() {
-        DatePickerElement localePicker = $(DatePickerElement.class)
-                .id("locale-change-picker");
-
-        setInputValueAndAssert(localePicker, "3/7/0900",
-                LocalDate.of(900, Month.MARCH, 7));
-
-        setInputValueAndAssert(localePicker, "3/6/900", "3/6/0900",
-                LocalDate.of(900, Month.MARCH, 6));
-        setInputValueAndAssert(localePicker, "3/5/0087",
-                LocalDate.of(87, Month.MARCH, 5));
-        setInputValueAndAssert(localePicker, "3/6/87", "3/6/1987",
-                LocalDate.of(1987, Month.MARCH, 6));
-        setInputValueAndAssert(localePicker, "3/7/20", "3/7/2020",
-                LocalDate.of(2020, Month.MARCH, 7));
-        setInputValueAndAssert(localePicker, "3/8/0020",
-                LocalDate.of(20, Month.MARCH, 8));
-
-        $("button").id("Locale-UK").click();
-        Assert.assertEquals("08/03/0020", localePicker.getInputValue());
-
-        setInputValueAndAssert(localePicker, "7/3/0900", "07/03/0900",
-                LocalDate.of(900, Month.MARCH, 7));
-
-        setInputValueAndAssert(localePicker, "6/3/900", "06/03/0900",
-                LocalDate.of(900, Month.MARCH, 6));
-        setInputValueAndAssert(localePicker, "5/3/0087", "05/03/0087",
-                LocalDate.of(87, Month.MARCH, 5));
-        setInputValueAndAssert(localePicker, "6/3/87", "06/03/1987",
-                LocalDate.of(1987, Month.MARCH, 6));
-        setInputValueAndAssert(localePicker, "7/3/20", "07/03/2020",
-                LocalDate.of(2020, Month.MARCH, 7));
-        setInputValueAndAssert(localePicker, "8/3/0020", "08/03/0020",
-                LocalDate.of(20, Month.MARCH, 8));
-
-        $("button").id("Locale-CHINA").click();
-        Assert.assertEquals("0020/3/8", localePicker.getInputValue());
-
-        setInputValueAndAssert(localePicker, "0900/3/7",
-                LocalDate.of(900, Month.MARCH, 7));
-        setInputValueAndAssert(localePicker, "900/3/6", "0900/3/6",
-                LocalDate.of(900, Month.MARCH, 6));
-        setInputValueAndAssert(localePicker, "0087/3/5",
-                LocalDate.of(87, Month.MARCH, 5));
-        setInputValueAndAssert(localePicker, "87/3/6", "1987/3/6",
-                LocalDate.of(1987, Month.MARCH, 6));
-        setInputValueAndAssert(localePicker, "20/3/7", "2020/3/7",
-                LocalDate.of(2020, Month.MARCH, 7));
-        setInputValueAndAssert(localePicker, "0020/3/8",
-                LocalDate.of(20, Month.MARCH, 8));
-
-        $("button").id("Locale-US").click();
-        Assert.assertEquals("3/8/0020", localePicker.getInputValue());
-    }
-
-    @Test
-    public void datePickerInsideDisabledParent_pickerIsDisabled() {
```

Comment:
These ITs were added in https://github.com/vaadin/vaadin-date-picker-flow/pull/59 when removing override of `isEnabled()` from the old version of component. None of components have such ITs so IMO they can be removed.